calico/_includes/content/calico-windows-install.md

```diff
@@ -0,0 +1,74 @@
+1. Download the {{site.prodnameWindows}} installation manifest.
+
+   ```bash
+{%- if include.networkingType == "vxlan" %}
+   curl {{ "/manifests/calico-windows-vxlan.yaml" | absolute_url }} -o calico-windows.yaml
+{%- else %}
+   curl {{ "/manifests/calico-windows-bgp.yaml" | absolute_url }} -o calico-windows.yaml
+{%- endif %}
+   ```
+
+1. Get the cluster's Kubernetes API server host and port, which will be used to update the {{site.prodnameWindows}} config map.
+   The API server host and port is required so that the {{site.prodnameWindows}} installation script can create a kubeconfig file for Calico services.
+   If your Windows nodes already have {{site.prodnameWindows}} installed manually, skip this step. The installation script will
+   use the API server host and port from your node's existing kubeconfig file if the `KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT` variables
+   are not provided in the `calico-windows-config` configmap.
+
+   {% include content/kube-apiserver-host-port.md %}
+
+1. Edit the `calico-windows-config` configmap in the downloaded manifest and ensure the required variables are correct for your cluster:
+   - `CALICO_NETWORKING_BACKEND`: This should be set to "vxlan".
```

Comment:
> Also what about install-reboot-enable remoteAccess service?

Thanks, yeah I missed some additional steps from the original install method for VXLAN too.

So this is yet another set of "prepare windows node" steps that the user will need to undertake. I don't know if we should handle this in `prepare-windows-node.ps1` otherwise the script will get complicated (it would need to handle a reboot). I think we should leave those steps to the user to manually run after running `prepare-windows-node.ps1`. WDYT?

Reply:
I've made the changes for now, please let me know what you think.

Reply:
Yeah, I don't think we can get this step into any script.